src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+import { Article } from './components/Article';
```

Comment:
Instead of using destructuring import, use default import for the Article component.